client/my-sites/domains/domain-management/list/empty-domains-list-card.jsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { Card } from '@automattic/components';
+import EmptyContent from 'calypso/components/empty-content';
+import { domainAddNew, domainUseYourDomain } from 'calypso/my-sites/domains/paths';
+import customerHomeIllustrationTaskFindDomain from 'calypso/assets/images/customer-home/illustration--task-find-domain.svg';
+import { isFreePlan } from '@automattic/calypso-products';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+function EmptyDomainsListCard( { selectedSite, hasDomainCredit, dispatchRecordTracksEvent } ) {
+	const translate = useTranslate();
+
+	const getActionClickHandler = ( type, buttonURL, sourceCardType ) => () => {
+		dispatchRecordTracksEvent( 'calypso_empty_domain_list_card_action', {
+			button_type: type,
+			button_url: buttonURL,
+			source_card_type: sourceCardType,
+		} );
+	};
+
+	const siteHasPaidPlan =
+		selectedSite?.plan?.product_slug && ! isFreePlan( selectedSite.plan.product_slug );
+
+	if ( siteHasPaidPlan && ! hasDomainCredit ) {
+		return null;
+	}
+
+	let title = translate( 'Get your domain' );
+	let line = translate( 'Get a free one-year domain registration or transfer with any paid plan.' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Get a free one-year domain registration with any paid plan.' )` *ES Score: 7*

Reply:
Are we supposed to? I think this is used in Gutenberg and for Calypso I don't see anything mentioning we should migrate to the `@wordpress/i18n` package.

Reply:
I'm not sure, but for example, most of the checkout flows now use it.